x-pack/plugins/ml/public/application/memory_usage/memory_usage_page.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState } from 'react';
+import { useTimefilter } from '@kbn/ml-date-picker';
+import { EuiFlexGroup, EuiFlexItem, EuiTabs, EuiTab } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { NodesList } from './nodes_overview';
+import { MlPageHeader } from '../components/page_header';
+import { JobMemoryTreeMap } from './memory_tree_map';
+import { useIsServerless } from '../contexts/kibana/use_is_serverless';
+
+enum TAB {
+  NODES,
+  MEMORY_USAGE,
+}
+
+export const MemoryUsagePage: FC = () => {
+  const serverless = useIsServerless();
+  const [selectedTab, setSelectedTab] = useState<TAB>(TAB.NODES);
+  useTimefilter({ timeRangeSelector: false, autoRefreshSelector: true });
+
+  return (
+    <>
+      <MlPageHeader>
+        <EuiFlexGroup responsive={false} wrap={false} alignItems={'center'} gutterSize={'m'}>
+          <EuiFlexItem grow={false}>
+            <FormattedMessage
+              id="xpack.ml.memoryUsage.memoryUsageHeader"
+              defaultMessage="Memory Usage"
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </MlPageHeader>
+
+      {serverless ? (
+        <JobMemoryTreeMap />
+      ) : (
+        <>
+          <EuiTabs>
```

Comment:
As discussed offline, this could do with some sort of info callout explaining what is being shown in this view, such as it only shows memory usage for open jobs?

Reply:
Added blue info callout in https://github.com/elastic/kibana/pull/149419/commits/69a48df2413c8fed84c22f098f96d2629b694975

@szabosteve do you have any suggestions for the text?

![image](https://user-images.githubusercontent.com/22172091/216581383-534e0a93-f553-45b8-a99e-f445a9c20231.png)